wger/settings_global.py

```diff
@@ -422,3 +422,7 @@
 
 SOCIAL_AUTH_GOOGLE_OAUTH2_KEY = os.getenv('SOCIAL_AUTH_GOOGLE_OAUTH2_KEY')
 SOCIAL_AUTH_GOOGLE_OAUTH2_SECRET = os.getenv('SOCIAL_AUTH_GOOGLE_OAUTH2_SECRET')
+
+FITBIT_CLIENT_ID="22CR5N"
```

Comment:
Let's have these set and extracted from the environment like the social auth keys and secrets above.